__fixtures__/test-project/package.json

```diff
@@ -20,8 +20,5 @@
   "prisma": {
     "seed": "yarn rw exec seed"
   },
-  "packageManager": "yarn@3.6.3",
-  "resolutions": {
-    "@types/react": "18.2.14"
-  }
+  "packageManager": "yarn@3.6.3"
```

Comment:
Will want to revertchanges to this file. Sorry about this - process is a bit manual at the moment.